### PR TITLE
docs: add naifu deploy guide

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: docs/.vitepress/config.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export default defineConfig({
         { text: '上下文过滤', link: '/advanced/filter' },
         { text: '自定义回复', link: '/advanced/i18n' },
         { text: '部署到公网', link: '/advanced/server' },
+        { text: '部署 NAIFU', link: '/advanced/naifu' },
```
Comment (shigma) [inline docs/.vitepress/config.ts:25]: 进阶部分讨论与 Koishi 相关的内容。建议此 PR 的内容专门开一个部分叫「自建」或者「部署」，放在进阶前面。

File: docs/advanced/naifu.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# 部署 NAIFU
+
+NAIFU 是匿名网友打包的一套 [NovelAI](https://novelai.net/) 的前端与后端应用，通过它可以快速搭建一个第三方的 NovelAI 服务。
+
+:::tip
```
Comment (shigma) [inline docs/advanced/naifu.md:5]:
```suggestion
::: tip
```

下同。

File: docs/advanced/naifu.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+透过磁力链接或网盘等方式下载 NAIFU，然后解压到一个目录中。
+
+如果你下载的内容中包含了 program.zip 文件，请将其解压到当前目录，不需要解压到子目录中。
+
+最终的目录结构应该如下所示：
+
+```text
+naifu/
+ |- clip/
```
Comment (shigma) [inline docs/advanced/naifu.md:20]:
```suggestion
├── clip/
```

底部可以使用 `└`。

File: docs/advanced/naifu.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+ |- requirements.txt
+ |- run.bat
+ |- run.sh
+ |- setup.bat
+ |- setup.sh
+ |- static/
+ |- taming/
+```
+
+## 安装（Windows）
```
Comment (shigma) [inline docs/advanced/naifu.md:36]:
```suggestion
## 安装 (Windows)
```

image

**Review by shigma — CHANGES_REQUESTED** [review]

File: docs/advanced/naifu.md (hunk 4 above)
Comment (MaikoTan) [reply docs/advanced/naifu.md]: You mean all full-width brackets should be changed to half-width ones?

File: docs/.vitepress/config.ts [hunk 5]
```diff
@@ -14,6 +14,11 @@ export default defineConfig({
         { text: '用法', link: '/usage' },
         { text: '配置项', link: '/config' },
       ],
+    }, {
+      text: '部署',
+      items: [
+        { text: '部署 NAIFU', link: '/delivery/naifu' },
```
Comment (shigma) [inline docs/.vitepress/config.ts:20]: I thought `部署` should be translated to `deploy`. 🤣 Does deliver really have this meaning?

File: docs/advanced/naifu.md (hunk 4 above)
Comment (shigma) [reply docs/advanced/naifu.md]: yes

File: docs/.vitepress/config.ts (hunk 5 above)
Comment (MaikoTan) [reply docs/.vitepress/config.ts:20]: Oops!
Yeah you're right, it should be `deploy`.
Recently I am working on an electric commerce platform app, maybe that is the reason. :rofl:

File: docs/deploy/naifu.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+::: tip
+如果你在启动 NAIFU 时指定了访问令牌，则在此处填入它。
+:::
+
+## 附录
+
+### NAIFU 下载地址
+
+- 原版磁力链接：magnet:?xt=urn:btih:4a4b483d4a5840b6e1fee6b0ca1582c979434e4d&dn=naifu&tr=udp%3a%2f%2ftracker.opentrackr.org%3a1337%2fannounce
```
Comment (shigma) [inline docs/deploy/naifu.md:119]:
```suggestion
- [磁力链接 (原版)](magnet:?xt=urn:btih:4a4b483d4a5840b6e1fee6b0ca1582c979434e4d&dn=naifu&tr=udp%3a%2f%2ftracker.opentrackr.org%3a1337%2fannounce)
```

File: docs/deploy/naifu.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+- 百度网盘：https://pan.baidu.com/s/1AoQgHf5UJuXB2qDtQkOVeQ?pwd=RA00 提取码：RA00
```
Comment (shigma) [inline docs/deploy/naifu.md:120]:
```suggestion
- [百度网盘](https://pan.baidu.com/s/1AoQgHf5UJuXB2qDtQkOVeQ?pwd=RA00)，提取码：RA00
```

**Comment by MaikoTan** [comment]
In fact, there is a tiny button named "add changes to batch", then you can apply the suggestion in just one commit...

**Comment by shigma** [comment]
soga :rofl:

---
Generated by [koishi-plugin-github](https://github.koishi.chat/).

**Pull request merged**